x-pack/plugins/security_solution/public/common/store/reducer.ts

```diff
@@ -113,13 +117,18 @@ export const createInitialState = (
  * Factory for the Security app's redux reducer.
  */
 export const createReducer: (
-  pluginsReducer: SubPluginsInitReducer
-) => Reducer<State, AppAction | AnyAction> = (pluginsReducer: SubPluginsInitReducer) =>
+  pluginsReducer: SubPluginsInitReducer,
+  tgridReducer: DataTableReducer // TODO: remove this param when the table reducer will be moved to security_solution
```

Comment:
I don't think it can be missed, but just in case can you link to this in one of the new issues you created in case someone else ends up doing the work?

Reply:
Overall the shape of the store will be changed when Timeline state will be moved to the timelines plugin.